### PR TITLE
Add more content around database configuration/setup

### PR DESCRIPTION
**Pull request by bradleybeddoes**

### CODE REVIEW

**Comment by bradleybeddoes** [comment]
@JakeSchoermer please review this and once you're happy this should help in tweaking ausaccessfed/validator-service#2 to help it pass PR.

**Comment by bradleybeddoes** [comment]
@JakeSchoermer it has been a week without you reviewing here...

**Comment by JakeSchoermer** [comment]
Thanks for the reminder.

Everything looks good to me, so I'm going to merge.

File: README.md [hunk 1]
```diff
@@ -414,14 +455,14 @@ RSpec.describe Permission, type: :model do
 end
 ```
 
-### Access Control
+## Access Control
 TODO
 
-### Controllers
```
Comment (JakeSchoermer) [inline README.md]: This is my fault, but since you merged my typo PR earlier, there is merge conflict in this branch, that will need to be resolved.

**Comment by bradleybeddoes** [comment]
@JakeSchoermer this is exactly why attending to PR which are assigned in a timely manner is important to us.

I have corrected the merge conflict that resulted here for re-review.

**Pull request merged**